.github/workflows/playground_examples_ci_reusable.yml

```diff
@@ -67,7 +67,7 @@ jobs:
           if [ -z "$BASE_REF" ] || [ "$BASE_REF" == "master" ]; then
             BASE_REF=origin/master
           fi
-          DIFF=$(git diff --name-only $BASE_REF $GITHUB_SHA | tr '\n' ' ')
+          DIFF=$(git diff --name-only --diff-filter=d $BASE_REF $GITHUB_SHA | tr '\n' ' ')
```

Comment:
If there are deleted files, wouldn't we still want to run the CI/CD step? Does it make more sense to put this logic in https://github.com/apache/beam/blob/master/playground/infrastructure/checker.py where we can create well defined logic for the deleted file case?

Reply:
Regardless, we'd probably want to filter on `dr`, not just `d`, right? https://git-scm.com/docs/git-diff

Reply:
Thank you Danny, I moved logic responsible for checking if file exists checker - there is really a potential scenario when deleting a file will break multi file example. As for filtering on [d] or [dr] it is still worth checking renamed files as this can also break examples. But the original issue was caused by trying to open deleted file.

Reply:
> As for filtering on [d] or [dr] it is still worth checking renamed files as this can also break examples. But the original issue was caused by trying to open deleted file.

Wasn't the original issue caused by a directory rename? https://github.com/apache/beam/issues/24807 A directory rename would show both files in the diff I think.

Regardless, the current approach of handling it in `checker.py` should work.

Reply:
Danny,  there were renamed files in Damon's PR  but there was also deleted file. I've tested git-diff behaviour locally for different sets of changed/renamed/deleted files and renames are reported as a "new file name" entry in git-diff output which was handled correctly in checker.py. It was a deleted file that caused the issue: https://github.com/apache/beam/pull/24806/files